app/static/src/scss/style.scss

```diff
@@ -76,6 +79,54 @@ $grey: #ccc;
   background-color: $light-grey;
 }
 
+.tooltip-error .tooltip-inner {
```

Comment:
All the tooltip styles could maybe be pulled out into their own scss file

Reply:
they have been now!